governance-core/src/json.rs

```diff
@@ -0,0 +1,232 @@
+use std::fs::File;
+use std::io::{self, BufReader, Error as IoError, ErrorKind as IoErrorKind, Read};
+use std::path::Path;
+
+use dusk_pki::{PublicKey, SecretKey};
+use rand::{rngs::StdRng, SeedableRng};
+use serde_json::Value;
+
+use crate::prelude::*;
+
+/// Parse a json file, convert them to a map of Transfers
+pub fn json_file<T: AsRef<Path>>(path: T) -> io::Result<TransferMap> {
+    let mut data = String::new();
+    let f = File::open(path.as_ref())?;
+
+    let mut reader = BufReader::new(f);
+    reader.read_to_string(&mut data)?;
+
+    json_bytes(data.as_bytes())
+}
+
+/// Parse raw json bytes convert them to a map of Transfers
+pub fn json_bytes<T: AsRef<[u8]>>(bytes: T) -> io::Result<TransferMap> {
+    let json: Value = serde_json::from_slice(bytes.as_ref())?;
+
+    if let Value::Object(obj) = json {
+        // Transfers holds all our transfers
+        let mut map = TransferMap::default();
+        // the account name and all events are the first key value pairs
+        for (account_name, events) in obj {
+            let events: Events = serde_json::from_value(events)?;
+            let events = events.events;
+            let from = public_key(account_name);
+
+            for Event {
+                occurrence,
+                changes,
+                cause,
+                ..
+            } in events
+            {
+                for Change { security, size, .. } in changes {
```

Comment:
Since the Reservation is not used, is better to skip for now
```suggestion
                for Change {
                    security,
                    size,
                    change_type,
                    ..
                } in changes
                {
                    if change_type == ChangeType::Reservation {
                        continue;
                    }
```